### PR TITLE
Implemented suggested article box

### LINKED ISSUE #60: Add suggested box to article sidebar
We want to have a sidebar widget that displays the top 5 articles from the current week.

### PR DESCRIPTION
**Pull request by psiemens**
Close #60 

Hey @annygakh, can you review this for me?

### CODE REVIEW

**Comment by annygakh** [comment]
Looks great!

File: ubyssey/templates/article/base.html [hunk 1]
```diff
@@ -45,9 +45,13 @@ <h1 class="headline">{{ article.headline|safe }}</h1>
                 {% else %}
                 <div class="sidebar">
                 {% endif %}
-                    <div class="advertisement box sidebar">
```
Comment (psiemens) [inline ubyssey/templates/article/base.html:48]: Why did you delete this? I'm just curious

**Review by psiemens — COMMENTED** [review]
This looks good, but I had a question about the missing ad
